### ISSUE TITLE
Userscript for CMC series controllers

### ISSUE THREAD

**Issue by strejda603** [steinbergmedia/midiremote-userscripts#16]
Hello.

Would it be possible to make Userscript for CMC controllers?

I know it's "Legacy Hardware" and you dropped support for it... But it will be nice, because old drivers aren't working anymore in Cubase 12 and they were really good pieces of hardware..

Thank you!

**Comment by jgarciaserra** [comment]
Yes, please. Great great piece off hardware and BEST integration ever
